src/Input/Factory/FieldFactory.php

```diff
@@ -58,15 +59,26 @@ public function create(
 
             $type = $schema->type;
             if (isset($schema->oneOf)) {
```

Comment:
it seems to me that you implemented "anyOf" logic, meaning it builds the first object that matches schema x response, but oneOf theoretically needs to validate that only 1 of the definitions is actually matching... should we maybe add support for both anyOf/oneOf in this PR and extend the logic when it's oneOf with an additional last step check about this "exactly one" constraint?